src/styles/_csemachine.scss

```diff
@@ -0,0 +1,3 @@
+.sa-cse-machine {
+  width: 0;
+}
```

Comment:
Where is this being used at?

Reply:
Not sure, but this file was already here so I've simply renamed this.

Reply:
It seems like it's applied to one of the `div` wrappers of the cse machine, so it should be safe to remove this file since applying `width: 0` does not seem to change anything.

Reply:
Great, if you've tested it and everything still seems to work fine, feel free to remove it, thanks!